docs/development/code_review.md

```diff
@@ -0,0 +1,81 @@
+
+## Verify that the PR solves the addressed issue
+
+### Understand the issue
+
+* Read the issue that was addressed by the PR and make sure you understand it
+* Read any discussions that occurred on the issue page
+
+### Check the Pull Request
+
+* Check the Pull Request for __completeness__: Was every point from the issue covered?
+* Check for __correctness__: Were the problems described in the issue solved in the intended way?
+* (For PRs that introduce new features:) Check the design - does it comply with our [guidelines][guidelines-general]?
+* Check for potential bugs - edge cases, exceptions that may be raised in functions that were called etc.
+* Check the code style: Is it readable? Do variables have sensible names? Is it consistent to existing code? Is there a better / more elegant way to do certain things (e.g. f-string instead of manual string concatenation)?
+* Check any warnings reported by your IDE
+
+## Check the tests
+
+* Run the tests locally
+* Check if there are any warnings that are not caught by the test suite
+* Make sure the test cases are complete - do they cover all edge cases (e.g. empty tables)?
+* Make sure they comply to our [guidelines][guidelines-tests] - are the tests parametrized and do all testcases have descriptive IDs?
+
+## Verify that the PR does not break existing code
+
+This is largely covered by the automated tests.
+However, you should always:
+
+* Make sure all tests actually ran through (pytest, linter, code coverage)
+* Make sure that the branch is up-to-date with the `main` branch, so you actually test the behaviour that will result once the feature branch is merged
+
+## Check the Pull Request format
+
+* Check that the PR title starts with a fitting [type](https://github.com/Safe-DS/.github/blob/main/.github/CONTRIBUTING.md#types) (e.g. `feat`, `fix`, ...)
+* Check that the changes introduced by the PR are documented in the PR message
+* Check that the `time spent by team` has been updated on the issue page
+
+## Requesting changes
+
+If you found any issues with the reviewed PR,
+navigate to the `Files changed` tab in the PR page
+and click on the respective line
+to add your comments.
+
+For more details on specific review features,
+check out the [documentation on GitHub][github-review].
+
+## Finishing the review
+
+When done, finish your review
+by navigating to the `Files changed` tab
+and clicking the green `Review changes` button
+on the top right.
+
+If you found no issues,
+select the `Approve` option
+to approve the changes made by the PR.
+If you like, you can add a "LGTM" meme
+from [lgtm.party](https://lgtm.party/),
+preferably one with a cat image.
+
+If no other reviewer suggested changes either,
+please also mark the corresponding issue
+as `Ready to merge` on the issue's page
+to update its status on the issue board.
+
+If you found any problems,
+select `Request Changes` instead.
+In this case, please also
+mark the corresponding issue
+as `In progress` again.
```

Comment:
Can be omitted since it was just neeeded for the PG.

Reply:
Removed it.